packages/core/src/utils/mixins/index.ts

```diff
@@ -7,7 +7,9 @@ export const getColor =
     const color =
       theme.colors[value][shade as keyof typeof theme.colors.Shading]
```

Comment:
I think we need to use optional chaining here. If the component isn't wrapped in a ThemeProvider, then `theme.colors[value][shade as keyof typeof theme.colors.Shading]` will throw an error because `theme.colors` won't exist.

We should do something like:
```ts
const color = theme.colors?.[value]?.[shade as keyof typeof theme.colors.Shading]
```